src/layouts/Dashboard/components/ReviewRequestModal/RequestOverview/RequestOverview.tsx

```diff
@@ -0,0 +1,335 @@
+import {
+  Avatar,
+  Text,
+  Box,
+  VStack,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+  HStack,
+  Icon,
+  Spacer,
+  Flex,
+  Link,
+  Breadcrumb,
+  BreadcrumbItem,
+  useTheme,
+} from "@chakra-ui/react"
+import {
+  Searchbar,
+  IconButton,
+  useSearchbar,
+} from "@opengovsg/design-system-react"
+import {
+  createColumnHelper,
+  flexRender,
+  getCoreRowModel,
+  Row,
+  useReactTable,
+} from "@tanstack/react-table"
+import {
+  BiGitCompare,
+  BiWorld,
+  BiFileBlank,
+  BiImage,
+  BiCog,
+  BiShow,
+  BiSort,
+  BiFilterAlt,
+  BiChevronRight,
+} from "react-icons/bi"
+import { TableVirtuoso } from "react-virtuoso"
+
+import { BxFileArchiveSolid } from "assets"
+
+interface DisplayedDateTime {
+  date: string
+  time: string
+}
```

Comment:
I think there are lots of types scattered around this file, would it be better if we moved them to their own file in the `types/` folder?

Reply:
can! what types do you foresee yourself using so that i can shift just those types?

Reply:
I won't be using these types! It was more of a matter of preference (separation of concerns) where we keep the declaration of types separate from the actual code logic.

Reply:
hmm in that case i **won't** export these - my reasoning is that i'd rather not place these in the shared folders as that implies that these would be used across the codebase and makes the codebase more difficult to navigate (encapsulation - private knowledge about this component shouldn't escape fi there's no need).  

having said that, if you do foresee yourself using this, please feel free to shift them to the `types/` folder!